src/lru_time_cache.rs

```diff
@@ -37,17 +37,14 @@ impl<K: Clone + Eq + Hash, V> LruTimeCache<K, V> {
     }
 
     /// Retrieves a reference to the value stored under `key`, or `None` if the key doesn't exist.
-    /// Also removes expired elements and updates the time.
     #[allow(dead_code)]
```

Comment:
```suggestion
```

Isn't `#[allow(dead_code)]` not needed as the function is public and hence isn't considered dead by clippy?